app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -25,9 +31,48 @@ function initializeBaseTracking() {
     }, 25);
     eventListening();
   }
-  trackCustomImpressions();
 }
 
+function trackGoogleAnalytics4() {
```

Comment:
SURPRISE! Google analytics v4 is a completely different script (gtag.js versus analytics.js) with different syntax. The upside is we can use both at the same time, but the downside is that you'll see I had to copy a lot of existing GAv3 code. 

This method is based on a great comment at ttps://stackoverflow.com/questions/22716542/google-analytics-code-explanation that breaks down how the old GA v3 async script loading work (Google doesn't provide such a script for GA4, they just expect you to hardcode it in)